relational/table.py

```diff
@@ -120,22 +120,42 @@ def _get_dbi_units(proj):
     """
     Returns:
       Net new units from DBI, only if it could be sourced from a new
-      construction permit.  None if no data from DBI.
+      construction permit or addition.  None if no data from DBI.
     """
     dbi_exist = 0
     dbi_prop = 0
     try:
-        dbi_exist = int(proj.field(
-            'existing_units', PTS.NAME,
-            entry_predicate=[_is_valid_dbi_type]))
+        fk_entries = proj.fields('existing_units',
+                                 PTS.NAME,
+                                 entry_predicate=[_is_valid_dbi_type])
+        for (fk, entries) in fk_entries.items():
+            latest = (None, datetime.min)
+            for entry in entries:
```

Comment:
Nit: I think this also has the de-dupe effect here fo removing dupe permit numbers right? I wonder if it's worth just adding a comment to this effect so its very clear (in case someone changes this code one day)